include/mbedtls/platform_util.h

```diff
@@ -68,14 +67,13 @@ void mbedtls_platform_zeroize( void *buf, size_t len );
 
 #if defined(MBEDTLS_HAVE_TIME_DATE)
 /**
- * \brief      Thread safe implementation of gmtime()
+ * \brief      Platform-specific implementation of gmtime_r()
  *
- *             The function is an abstraction that when called behaves similar
- *             to the gmtime() function from the C standard, but is thread
- *             safe.
+ *             The function is a thread-safe abstraction that behaves
+ *             similar to the gmtime_r() function from the C standard.
```

Comment:
This paragraph was originally written for `gmtime` and doesn't quite work for `gmtime_r`. `gmtime_r` is not from the C standard. This function doesn't just “behave similar[ly]” to `gmtime_r`, it has the semantics of `gmtime_r`. Suggestion:

> `\brief` Platform-specific implementation of gmtime_r()

> This function behaves similarly to the gmtime() function from the C standard, but is thread-safe. It has the same semantics as the gmtime_r() function from Unix/POSIX.

Reply:
I don't like to say that it is similar to `gmtime` because looking at its signature and semantics, it isn't, and implementation details shouldn't matter at this level. What about the following: 
```
\brief Platform-specific implementation of gmtime_r()
The function is a thread-safe abstraction that behaves
similar to the gmtime_r() function from Unix/POSIX.
...
```

Reply:
Fixed in 03b2bd4.

Reply:
It is similar, but not identical, to `gmtime`: it calculates the same value, but in a different way.

Reply:
Agreed, but  _how_ the function achieves what it promises shouldn't be (a prominent) part of the documentation.